pkg/storage/gcs/gcs.go

```diff
@@ -320,7 +326,8 @@ func (d *driver) CreateStorage(cr *imageregistryv1.Config) error {
 				cr.Spec.Storage.GCS = d.Config.DeepCopy()
 			}
 		}
-		return addTagsToStorageBucket(d.Context, cr, d.Listers, d.Config.Bucket, d.Config.Region)
+		// add user-defined tags to the created storage bucket and update `StorageTagged` condition.
+		return updateTagCondition(cr, tagMgr.AddTagsToStorageBucket(d.Context, cr))
```

Comment:
What do you think of separating this into two calls? I think it would be easier to read. i.e

```suggestion
		err := tagMgr.AddTagsToStorageBucket(d.Context, cr)
		return updateTagCondition(cr, err)
```